src/routes/ResultsOverviewScreen.tsx

```diff
@@ -117,6 +117,7 @@ interface ResultOverviewScreenPropsType {
 
 const ResultsOverview = ({ navigation, route }: ResultOverviewScreenPropsType): ReactElement => {
   const { exercise, results, discipline } = route.params.result
+  const { repeatExercise } = route.params
```

Comment:
AFAIK functions should not be passed as route params, better to just pass the necessary values or just duplicate the function. This is also indicated by the warning: `Non-serializable values were found in the navigation state.` While it works, it is not good practice. See https://reactnavigation.org/docs/troubleshooting/#i-get-the-warning-non-serializable-values-were-found-in-the-navigation-state

Reply:
I see, thank you. Wanted to avoid copying the function, but will do, in this case.